src/Storage/Storage.Management/help/New-AzDataLakeGen2Item.md

```diff
@@ -0,0 +1,348 @@
+```
+
+This command creates a folder with specified Permission, Umask, properties, and metadata
+
+### Example 2: Create(upload) a data lake file from a local source file, and the cmdlet runs in background
+```
+PS C:\> $task = New-AzDataLakeGen2Item  -Container "testcontainer" -Path "dir1/dir2/file1" -Source "c:\sourcefile.txt" -Force -asjob
+PS C:\> $task | Wait-Job
+PS C:\> $task.Output
+
+   Container Uri: https://storageaccountname.blob.core.windows.net/container1
+
+Path                 IsDirectory  Length          ContentType                    LastModified         Permissions  Owner      Group               
+----                 -----------  ------          -----------                    ------------         -----------  -----      -----               
+dir1/dir2/file1      False        14400000        application/octet-stream       2019-10-29 09:04:44Z
+```
+
+This command creates(upload) a data lake file from a local source file, and the cmdlet runs in background.
+
+## PARAMETERS
+
+### -AsJob
+Run cmdlet in the background
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ClientTimeoutPerRequest
+The client side maximum execution time for each request in seconds.
+
+```yaml
+Type: System.Nullable`1[System.Int32]
+Parameter Sets: (All)
+Aliases: ClientTimeoutPerRequestInSeconds
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ConcurrentTaskCount
+The total amount of concurrent async tasks.
+The default value is 10.
+
+```yaml
+Type: System.Nullable`1[System.Int32]
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Container
+Container name
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -Context
+Azure Storage Context Object
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.IStorageContext
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName, ByValue)
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Folder
+Indicates that this new item is a folder and not a file.
```

Comment:
May add "Without specifying this parameter, it's a file item".

Reply:
added